tutorial/README.md

```diff
@@ -150,16 +140,7 @@ link by appending an `image_url` query parameter:
 ### Final Note on File Formats and OME-Zarr
 
 The Glencoe software and OME teams hava been clear that the "raw" N5/Zarr formats produced by `bioformats2raw` should be considered
-experimental for the time being as intermediates for generating valid OME-TIFFs. Therefore `Option 1` is not as stable as `Option 2`
-for generating images for Avivator/Viv.
-
-However, there is activate community development for a next generation file format (NGFF) called
-[OME-Zarr](https://github.com/ome/omero-ms-zarr/blob/master/spec.md), which can be produced in part by
-running `bioformats2raw --file_type=zarr --dimension-order='XYZCT'`. This will generate a valid multiscale Zarr
-which is compatible with OME-Zarr, but is missing some metadata within the Zarr hierarchy.
-
-Aviviator can view the "raw" output as described above, and the _same_ multiscale pyramid can also be viewed
-in desktop analysis tools like [`napari`](https://github.com/napari/napari).
+experimental for the time being as intermediates for generating valid OME-TIFFs. Therefore using these intermediate outputs directly for viewing is not recommended.
```

Comment:
Think we probably don't need this either. The start of the tutorial explains that the intermediate should not be used
```suggestion
```